README.md

```diff
@@ -32,21 +32,28 @@ make testnet-up
-| SimpleAccount         | 0x661b4a3909b486a3da520403ecc78f7a7b683c63 | Balance: 100 ether            |
-| Counter               | 0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9 |                               |
-| PasskeyAccountFactory | 0x5FC8d32690cc91D4c39d9d3abcBD16989F875707 |                               |
-| PasskeyAccount        | **_Depending on PASSKEY env variables_**   | Balance: 100 ether            |
-| VerifyingPaymaster    | 0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6 | EntryPoint deposit: 100 ether |
+- Deployer 1: Non-account abstraction contracts
 
-> If you use default passkey, the PasskeyAccount address would be `0xf30a89a6a3836e2b270650822e3f3cebff3b7497`.
+| Contract | Address                                    |
+| -------- | ------------------------------------------ |
+| Counter  | 0x8464135c8F25Da09e49BC8782676a84730C318bC |
 
-The owner of these contracts is `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266`, which is the first address derived from testing mnemonic `test test test test test test test test test test test junk`, and it has unlimited balance of ether.
+- Deployer 2: Account abstraction v0.6.0 contracts
 
-> Please make sure the contracts in the submodules(`lib`) are added into `src/Artifacts.sol` for being compiled.
+| Contract              | Address                                    | Note                        |
+| --------------------- | ------------------------------------------ | --------------------------- |
+| EntryPoint            | 0x663F3ad617193148711d28f5334eE4Ed07016602 |                             |
+| SimpleAccountFactory  | 0x2E983A1Ba5e8b38AAAeC4B440B9dDcFBf72E15d1 |                             |
+| SimpleAccount         | 0x1E684E8937774B00Ee2Ea562256f27a5c9D20d7c | Balance: 100 ETH            |
+| PasskeyAccountFactory | 0xBC9129Dc0487fc2E169941C75aABC539f208fb01 |                             |
+| PasskeyAccount        | **_Depends on PASSKEY env variables_**     | Balance: 100 ETH            |
+| VerifyingPaymaster    | 0xF6168876932289D073567f347121A267095f3DD6 | EntryPoint deposit: 100 ETH |
+
+> If you use default passkey, the PasskeyAccount address would be `0xF4bb6e38fC8A5ec977D4Fdc74B4E0fa84c8dc704`.
+
+These contracts are mainly owned by `0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266`. For easier management, deployer 1 (`0x70997970C51812dc3A010C7d01b50e0d17dc79C8`) handles Counter and non-account abstraction contracts. Deployer 2 (`0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC`) handles account abstraction v0.6.0 contracts. Deployer 3 (`0x90F79bf6EB2c4f870365E785982E1f101E93b906`) handles account abstraction v0.7.0 contracts. Owner and deployers are the first four addresses from the mnemonic `test test test test test test test test test test test junk`, and the first address has unlimited balance of ether.
+
+> Please make sure the contracts in the submodules(`lib`) are added into `src/artifacts/0.6/Artifacts.sol` or `src/artifacts/0.6/Artifacts.sol` for being compiled.
```

Comment:
```suggestion
> Please make sure the contracts in the submodules(`lib`) are added into `src/artifacts/{0.6, 0.7}/Artifacts.sol` for being compiled.
```

Reply:
Considering shell compatibility, would it be better to use `src/artifacts/{0.6,0.7}/Artifacts.sol` (without spaces in between)?